client/my-sites/email/email-providers-stacked-comparison/index.jsx

```diff
@@ -0,0 +1,470 @@
+				}
+				const { errors } = this.props?.cart?.messages;
+				if ( errors && errors.length ) {
+					// Stay on the page to show the relevant error(s)
+					return;
+				}
+				this.isMounted && page( '/checkout/' + selectedSiteSlug );
+			} );
+	};
+
+	onGoogleUsersChange = ( changedUsers ) => {
+		this.setState( { googleUsers: changedUsers } );
+	};
+
+	onGoogleFormReturnKeyPress = ( event ) => {
+		// Simulate form submission
+		if ( event.key === 'Enter' ) {
+			this.onGoogleConfirmNewUsers();
+		}
+	};
+
+	onGoogleConfirmNewUsers = () => {
+		const { googleUsers } = this.state;
+
+		const usersAreValid = areAllUsersValid( googleUsers );
+
+		recordTracksEvent( 'calypso_email_providers_add_click', {
+			mailbox_count: googleUsers.length,
+			mailboxes_valid: usersAreValid ? 1 : 0,
+			provider: 'google',
+		} );
+
+		if ( ! usersAreValid ) {
+			return;
+		}
+
+		const { domain, productsList, selectedSiteSlug, shoppingCartManager } = this.props;
+		const domains = [ domain ];
+		const googleProductSlug = config.isEnabled( 'google-workspace-migration' )
+			? GOOGLE_WORKSPACE_BUSINESS_STARTER_YEARLY
+			: GSUITE_BASIC_SLUG;
+
+		this.setState( { addingToCart: true } );
+		shoppingCartManager
+			.addProductsToCart(
+				getItemsForCart( domains, googleProductSlug, googleUsers ).map( ( item ) =>
+					fillInSingleCartItemAttributes( item, productsList )
+				)
+			)
+			.then( () => {
+				if ( this.isMounted ) {
+					this.setState( { addingToCart: false } );
+				}
+				const { errors } = this.props?.cart?.messages;
+				if ( errors && errors.length ) {
+					// Stay on the page to show the relevant error(s)
+					return;
+				}
+				this.isMounted && page( '/checkout/' + selectedSiteSlug );
+			} );
+	};
+
+	renderEmailForwardingCard() {
+		const { domain, translate } = this.props;
+
+		const buttonLabel =
+			domain.emailForwardsCount > 0
+				? translate( 'Manage email forwarding' )
+				: translate( 'Add email forwarding' );
+
+		return (
+			<EmailProviderCard
+				providerKey="forwarding"
+				logo={ { path: forwardingIcon } }
+				title={ translate( 'Email Forwarding' ) }
+				description={ translate(
+					'Use your custom domain in your email address and forward all your mail to another address.'
+				) }
+				detailsExpanded={ this.state.expanded.forwarding }
+				onExpandedChange={ this.onExpandedStateChange }
+				buttonLabel={ buttonLabel }
+				expandButtonLabel={ translate( 'Add email forwarding' ) }
+				onButtonClick={ this.goToEmailForwarding }
+				features={ getEmailForwardingFeatures() }
+			/>
+		);
+	}
+
+	renderGoogleCard() {
+		const { currencyCode, domain, gSuiteProduct, isGSuiteSupported, translate } = this.props;
+
+		// TODO: Improve handling of
+		if ( ! isGSuiteSupported ) {
+			return null;
+		}
+
+		const logoPath = config.isEnabled( 'google-workspace-migration' )
+			? googleWorkspaceIcon
+			: gSuiteLogo;
+
+		const formattedPrice = translate( '{{price/}} /user /month billed annually', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **3** times:
`translate( 'per month{{br/}}billed yearly' )` *ES Score: 6*